src/versionize/src/primitives.rs

```diff
@@ -61,6 +63,37 @@ impl_versionize!(f32);
 impl_versionize!(f64);
 impl_versionize!(char);
 impl_versionize!(String);
+impl_versionize!(AtomicUsize);
```

Comment:
As per discussion, let's not impl Versionize for Atomics.

Reply:
Done.